sdk/src/native_loader.rs

```diff
@@ -1,13 +1,46 @@
 use crate::{account::Account, hash::Hash};
+use num_derive::FromPrimitive;
 
 crate::declare_id!("NativeLoader1111111111111111111111111111111");
 
+#[derive(Debug, Clone, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct Info {
+    pub kind: Kind,
+    pub name: String,
+}
+#[derive(Debug, Clone, Copy, Deserialize, Eq, FromPrimitive, Hash, PartialEq, Serialize)]
+pub enum Kind {
+    Program = 1,
+    Loader = 2,
+}
+
+#[macro_export]
+macro_rules! native_program_info(
+    ($name:expr) => (
+        $crate::native_loader::Info{
+            kind: $crate::native_loader::Kind::Program,
+            name: $name.to_string(),
+        }
+    )
+);
+#[macro_export]
+macro_rules! native_loader_info(
+    ($name:expr) => (
+        $crate::native_loader::Info{
```

Comment:
```suggestion
        $crate::native_loader::Info {
```

Reply:
Thanks, it sucks that `fmt` doesn't always fixup macros